<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>三菱机器人选型软件</string>
  </property>
  <widget class="QComboBox" name="tMod">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>69</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>四轴</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>六轴</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>机器人类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>54</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>负重</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>工作半径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bChoose">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>54</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>MM</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tResult">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>291</width>
     <height>691</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>2</height>
    </size>
   </property>
   <property name="rowCount">
    <number>20</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>250</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>25</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>28</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>25</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>机器人型号</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tDetail1">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>281</width>
     <height>601</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>2</height>
    </size>
   </property>
   <property name="rowCount">
    <number>18</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>170</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>25</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>25</number>
   </attribute>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string>型号</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>环境规格</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>防护等级</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>安装姿势</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>构造</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>动作自由度</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>驱动方式</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>位置检测方式</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>可搬运重量</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>臂长</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>最大工作半径</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>动作范围J1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>动作范围J2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>动作范围J3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>动作范围J4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>动作范围J5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>动作范围J6</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>参数</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tDetail2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>120</y>
     <width>281</width>
     <height>601</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>2</height>
    </size>
   </property>
   <property name="rowCount">
    <number>19</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>170</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>25</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>25</number>
   </attribute>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string>最大速度J1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>最大速度J2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>最大速度J3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>最大速度J4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>最大速度J5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>最大速度J6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>最大合成速度</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>周期时间</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>位置重复精度</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>环境温度</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>本体重量</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>允许力矩J4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>允许力矩J5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>允许力矩J6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>允许惯量J4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>允许惯量J5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>允许惯量J6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>最大惯量</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>参数</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tDetail3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>730</y>
     <width>561</width>
     <height>181</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>2</height>
    </size>
   </property>
   <property name="rowCount">
    <number>5</number>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>400</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>25</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>25</number>
   </attribute>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string>工具配线</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>工具气动配管</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>设备间电缆</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>连接控制器</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>参数</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>54</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>KG</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>110</y>
     <width>561</width>
     <height>381</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../微信图片_20180412172601.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="tLoad">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="tR">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>530</y>
     <width>541</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>惯量计算</string>
   </property>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>惯量计算结果</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tL2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>半径</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tR10">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="tI">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>L1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>140</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tL1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tA1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>半径</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>21</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>长</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>宽</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bCalc">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>计算惯量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>重量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tR11">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="tB1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>抓手形状</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tA">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="tM1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="tRectangle">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>方形</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>圆形</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>工件形状</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tB">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>KG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>长</string>
    </property>
   </widget>
   <widget class="QComboBox" name="tRectangle1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>方形</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>圆形</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>L2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>KG</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>21</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>宽</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tM">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>重量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bCalc2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>加入筛选</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
